Sources/Apollo/InterceptorProvider.swift

```diff
@@ -65,6 +56,10 @@ open class LegacyInterceptorProvider: InterceptorProvider {
         LegacyCacheWriteInterceptor(store: self.store),
     ]
   }
+  
+  open func additionalErrorInterceptor<Operation>(for operation: Operation) -> ApolloErrorInterceptor? where Operation : GraphQLOperation {
```

Comment:
Do you mind changing this and below to the `<Operation: GraphQLOperation>` instead of the `where` clause to match format in the rest of the code?

Reply:
done! 👍